src/aks-preview/azext_aks_preview/commands.py

```diff
@@ -276,3 +278,28 @@ def load_command_table(self, _):
             'aks_mesh_disable_egress_gateway',
             supports_no_wait=True,
             confirmation=True)
+        g.custom_command(
+            'get-revisions',
+            'aks_mesh_get_revisions',
+            table_transformer=aks_mesh_revisions_table_format)
+        g.custom_command(
+            'get-upgrades',
+            'aks_mesh_get_upgrades',
+            table_transformer=aks_mesh_upgrades_table_format)
```

Comment:
It is better to add tests for these two new added commands.

Reply:
@yanzhudd 
Added live tests for both these commands. please take a look.|